scripts/pylib/twister/twisterlib/runner.py

```diff
@@ -1179,6 +1180,8 @@ def cmake_assemble_args(extra_args, handler, extra_conf_files, extra_overlay_con
         args_expanded.extend(["-D{}".format(a.replace('"', '\"')) for a in cmake_extra_args])
         args_expanded.extend(["-D{}".format(a.replace('"', '')) for a in args])
 
+        args_expanded.append('-DWEST_PYTHON={}'.format(pathlib.Path(sys.executable).as_posix()))
```

Comment:
@tejlmand, should it be `Python3_EXECUTABLE`  ? #72232
```suggestion
        args_expanded.append('-DPython3_EXECUTABLE={}'.format(pathlib.Path(sys.executable).as_posix()))
```

Reply:
> @tejlmand, should it be Python3_EXECUTABLE ? #72232

yes it should. `WEST_PYTHON` is intended for `west`.